price-feeder/oracle/provider/mock.go

```diff
@@ -118,3 +118,12 @@ func (p MockProvider) GetCandlePrices(pairs ...types.CurrencyPair) (map[string][
 func (p MockProvider) SubscribeCurrencyPairs(pairs ...types.CurrencyPair) error {
 	return nil
 }
+
+// GetAvailablePairs return all available pairs symbol to susbscribe.
+func (p MockProvider) GetAvailablePairs() (map[string]struct{}, error) {
+	return map[string]struct{}{
+		"UMEEUSDT": {},
+		"ATOMUSDC": {},
+		"ATOMUSDT": {},
+	}, nil
```

Comment:
We could grab these values from the spreadsheet, but since that doc will probably never change this should be fine

Reply:
done at https://github.com/umee-network/umee/pull/649/commits/12582ff58f81243a68120c84eaa91818d211af2d